scripts/ssl.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#################################################################
+#                                                               #
+# Script By Vasileios Ntoufoudis                                #
+# info@ntoufoudis.com                                           #
+#                                                               #
+#################################################################
```

Comment:
Vasileios, what about instead of attributing your contribution here we have you push from your own account and do a new MR, so you get proper credit where credit is due?

Another thought, how about if you want to put your contact info into the code, we put it into the actual README.md starter page rather than embedded within the code? I want to ensure there is not confusion that this code is indeed MIT open source.

Reply:
Already resolved.